docs/program/extend/SDK-as-server.md

```diff
@@ -11,41 +11,61 @@ videoAlt: "A quadrupedal robot comprised of small servos, black laser cut acryli
+
 Instructions on creating and using modular resources are available [here](/program/extend/modular-resources).
 {{% /alert %}}
 
-If a type or model of [component](/components) you are working with is not built into the [Viam RDK](/internals/rdk), you can use a [Viam SDK](/program/sdk-as-client) to code a custom resource implementation, host it on a server, and add it as a [remote](/manage/parts-and-remotes) of your robot.
-Then, you can configure a command to launch this remote server as a [process](/appendix/glossary/#term-process) of your robot to make sure the remote server is always running alongside the rest of your robot.
+If a type or model of [component](/components) you are working with is not built-in to the [Viam RDK](/internals/rdk), you can use a [Viam SDK](/program/sdk-as-client) to code a custom resource implementation, host it on a server, and add it as a [remote](/manage/parts-and-remotes) of your robot.
 
-After configuring the remote server, you can control and monitor your component programmatically with the SDKs and from the [Viam app](https://app.viam.com/).
+After configuring the remote server, control and monitor your component programmatically with the SDKs and from the [Viam app](https://app.viam.com/).
 
-For example, let's say that you have a robotic arm that is not one of the models supported by [Viam's arm component](/components/arm/), and you want to integrate it with Viam.
-You will need to create a custom component and register the new arm model in order to use it with the Viam SDK.
-Once your new arm is registered, you will be able to use it remotely with Viam.
+For example:
 
-{{% alert title="Tip" color="tip" %}}
-Here is an example of [how to create a custom arm component in the Python SDK documentation](https://python.viam.dev/examples/example.html#subclass-a-component).
-{{% /alert %}}
+- You have a robotic arm that is not one of the models supported by [Viam's arm component](/components/arm/), and you want to integrate it with Viam.
+- You create a custom component and register the new arm model with a Viam SDK.
+- You control it remotely as part of your robot.
+
+This example is available in the [Python SDK documentation](https://python.viam.dev/examples/example.html#subclass-a-component).
+
+## Instructions
+
+To add a custom resource as a [remote](/manage/parts-and-remotes):
 
-To add a custom resource as a remote:
+{{< tabs >}}
+{{% tab name="Go" %}}
 
-1. Subclass a component and implement desired functions.
+1. Code a new model of a built-in resource type. You can do this by creating a new interface that implements required methods. The new model must implement any functions of the built-in resource type marked as required in its RDK API definition.
+2. Register the custom component on a new gRPC server instance and start the server.
+You can do this with the `go.viam.com/utils/rpc` package.
```

Comment:
Probably don't want people directly using that. In most cases with go, I expect they'd create a local robot to be a server, using the functions in RDK itself.

Reply:
Gotcha, do you think this is now illustrative enough? Or should I point that out directly -- "create a local robot as a server"

Reply:
I think that's good enough until/unless you're going to do a MUCH more in depth set of step-by-step instructions (more like a tutorial.)